cmd/aws-janitor-boskos/main.go

```diff
@@ -35,6 +36,7 @@ import (
 
 var (
 	boskosURL          = flag.String("boskos-url", "http://boskos", "Boskos URL")
+	rTypes             common.CommaSeparatedStrings
```

Comment:
maybe set the default value here instead?

Reply:
When I set the default value here, I hit an error due to [this] check(https://github.com/kubernetes-sigs/boskos/blob/ae3497d24cd75fef284c9d3c766d63a6b2416a58/common/common.go#L211)

```
./_output/bin/aws-janitor-boskos -resource-type=aws-account,abc-aws-account
invalid value "aws-account,abc-aws-account" for flag -resource-type: resTypes flag already set
```